src/widget.h

```diff
@@ -104,6 +105,7 @@ class BaseWidget {
 	uint16 fill_y;             ///< Vertical fill step.
 	uint16 resize_x;           ///< Horizontal resize step.
 	uint16 resize_y;           ///< Vertical resize step.
+	bool repeating_events;     ///< Receive repeating events.
```

Comment:
Name doesn't mean much to me. Explanation doesn't clarify it either.

What is a "repeating event"? Getting the same event multiple times? To me, a press event and a release event are not the same, if that is what you mean. They are significantly different, eg in drag detection.